.changeset/orange-clocks-sin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/presentation-hierarchies-react": patch
```

Comment:
This should be a minor change since API was broken. Also could you mention that this breaks existing API and how code should be adjusted?

Reply:
Changed to minor and provided more information.